event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/CollectionExercise.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.ons.ctp.common.event.model;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.util.Date;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.ons.ctp.common.jackson.CustomDateSerialiser;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class CollectionExercise implements EventPayload {
+  private String collectionExerciseId;
+  private String surveyId;
+  private String name;
+  private String reference;
+
+  @JsonSerialize(using = CustomDateSerialiser.class)
```

Comment:
where is the requirement for the start/end and metadata ?

Reply:
Merged with main and added unit tests